tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/ProgrammeMembershipServiceImpl.java

```diff
@@ -180,8 +183,8 @@ public ProgrammeMembershipDTO findOne(Long id) {
   }
 
   /**
-   * Delete the curriculumMembership by id.
-   * NOTE: this is the curriculumMembership, not the containing programmeMembership
+   * Delete the curriculumMembership by id. NOTE: this is the curriculumMembership, not the
```

Comment:
There are a few things in this method that I'd like to pick up to make it less likely to be misused, including standardising a delete for programme memberships.
Can we talk next week about whether a second refactoring (following this PR) would be useful?